inventories/local/hosts

```diff
@@ -12,13 +12,13 @@ frontend ansible_host=192.168.60.19 ansible_user=vagrant ansible_ssh_private_key
 validator0[1:5]
 
 [avalanche_nodes:children]
-bootstrap_node
+bootstrap_nodes
 subnet_txs_host
 subnet_validators
 api_node
 
-[bootstrap_node]
-validator01
+[bootstrap_nodes]
+validator0[1:2]
```

Comment:
We should not have validator01 and validator02 since we only have `[NodeID-7Xhw2mDxuDS44j42TCB6U5579esbSt3Lg]` as default for `avalanchego_bootstrap_node_ids` at https://github.com/AshAvalanche/ansible-avalanche-collection/pull/84

This situation results in this error:

```
couldn't load node config: expected the number of bootstrapIPs (2) to match the number of bootstrapIDs (1)
```